common/test/acceptance/tests/lms/test_learner_profile.py

```diff
@@ -491,10 +491,22 @@ def test_user_can_see_error_for_wrong_file_type(self):
         self.assert_default_image_has_public_access(profile_page)
 
         profile_page.upload_file(filename='generic_csv.csv')
-        self.assertEqual(
-            profile_page.profile_image_message,
-            "The file must be one of the following types: .gif, .png, .jpeg, .jpg."
+        self.assertIn(
+            "The file must be one of the following types:", profile_page.profile_image_message,
```

Comment:
Can we just fix the message to sort the extensions instead?  The code for that is [here](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/profile_images/images.py#L230); just update it to produce a list of the extensions, sort that, and then convert to a comma-separated string.

Reply:
sure I will update accordingly.

Reply:
This method is not effecting the test case.